CloudinaryManager.js

```diff
@@ -0,0 +1,58 @@
+export default class CloudinaryManager {
+    constructor(options) {
+        this.options = options;
+    }
+
+    createMediaString(formdata) {
+        var mediaStr;
+
+        if (formdata.minWidth !== "") {
+            mediaStr = `media="(min-width: ${formdata.minWidth})"`;
+        } else if (formdata.maxWidth !== "") {
+            mediaStr = `media="(max-width: ${formdata.maxWidth})"`;
+        }
+
+        return mediaStr;
+    }
+
+    createImageString(formdata) {
+        let acctName = "dnginndga";
```

Comment:
Remove acctName and use options instead.

Reply:
fixed